### PR TITLE
Add randomchoice function that picks a random value

### LINKED ISSUE #1907: Command variable that returns a random outcome (coinflip)
It would be nice to have a command variable that can pick between two outcomes at random.

Example: $(args:0-1) was ejected! He was $(coinflip("not"|" ")) the imposter amongE ❗

### PR DESCRIPTION
**Pull request by pajlada**
NOTE: This updates the argument matching regex - I've done by best to
ensure this does not break anything by adding tests in previous PRs

Fixes #1907

Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [x] I have tested all changes

### CODE REVIEW

File: pajbot/bot.py [hunk 1]
```diff
@@ -60,6 +60,8 @@
 
 SLICE_REGEX = re.compile(r"(-?\d+)?(:?(-?\d+)?)?")
 
+RANDOMCHOICE_ARGUMENT_REGEX = re.compile(r"(\"(\w+)\",? ?)")
```
Comment (hotbear1110) [inline pajbot/bot.py:63]:
```suggestion 
	RANDOMCHOICE_ARGUMENT_REGEX = re.compile(r"(\"(\"(((|\s)(\w+|)(\s|))+)\",? ?)")
 ```

**Review by hotbear1110 — CHANGES_REQUESTED** [review]

File: pajbot/bot.py (hunk 1 above)
Comment (hotbear1110) [reply pajbot/bot.py:63]: I just tested this and seems to be working fine. Only limitation is non-word characters like periods or commas in a sentence.

**Comment by ALazyMeme** [comment]
Would we not be better off making random an object i/o having randomchoice as a specific variable? Could add more options onto it:
- number
- chatter
- emote

Something like that.

File: pajbot/bot.py (hunk 1 above)
Comment (pajlada) [reply pajbot/bot.py]: Resolved in [`a01d4a4` (#1920)](https://github.com/pajbot/pajbot/pull/1920/commits/a01d4a4e5d3c778283252349d400f2f8c91a7b8d)

**Comment by pajlada** [comment]
> Would we not be better off making random an object i/o having randomchoice as a specific variable? Could add more options onto it:
> 
>     * number
> 
>     * chatter
> 
>     * emote
> 
> 
> Something like that.

More functions can be added like `$(randomemote)` or `$(randomrange)`

**Review by ALazyMeme — APPROVED** [review]

**Pull request merged**